pages/about.html

```diff
@@ -1,12 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/about.css">
+    <title>Document</title>
 </head>
+
 <body>
-  
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo">ABOUT</div>
+            <ul class="nav">
+                <li>
+                    <a href="index.html">Home</a>
+                </li>
+                <li>
+                    <a href="portfolio.html">portfolio</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <header class="header ">
+        <div class="container ">
+            <div>
+                <h1>Education & Work Experience</h1>
+                <p>About My Education and Work Experiences</p>
+            </div>
+            <img src="../image/vida.png" alt="Vida Maleki">
+        </div>
+    </header>
+    <section class="boxes ">
+        <div class="container ">
+            <div class="box ">
+                <h2>University</h2>
+                <p>Bachelor's degree in accounting</p>
+                <p>Islamic Azad University of Tabriz</p>
+                <p>From 2009 to 2014</p>
+            </div>
+            <div class="box ">
+                <h2>YWCA</h2>
+                <p>Bank$Work certificate in 2018</p>
+            </div>
+            <div class="box ">
+                <h2>Ada Developer Academy</h2>
+                <p>Full stack developer 2022</p>
+            </div>
+            <div class="box ">
+                <h2>Zarin Naghsh</h2>
+                <p>sales superviser - </p>
+                <p>accounting assistant</p>
+            </div>
+            <div class="box ">
+                <h2>KeyBank</h2>
+                <p>Teller-6 months</p>
+                <p>Lead Teller-9 months</p>
+                <p>2018-2019</p>
+            </div>
+            <div class="box ">
+                <h2>Umpqua Bank</h2>
+                <p>Universal Assosiate- 1 year</p>
+                <p>2020</p>
+            </div>
+        </div>
+    </section>
+    <footer class="footer">
+        <p>&copy;2022;Vida Maleki</p>
+    </footer>
```

Comment:
We probably don't need the class here - we can just refer to the `footer` tag in our CSS.